js/carousel.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 Benjamin W. Bohl. Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ * http://www.osedu.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/*jslint browser:true*/
+
+var intervals = [];
+
+/*
+ * sets an interval timer for a submitted carousel
+ * @param {item} carousel - HTMLcolleciton item.
```

Comment:
The "submitted carousel" is a call to `document.getElementById` which actually returns an Element object (https://developer.mozilla.org/en-US/docs/Web/API/Element). One could specify it as HTMLElement (https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement) since we do not expect any other element types here?

Reply:
definitely right, as HTMLCollection also is defined as array of HTMLElement:s

```suggestion
 * @param {Object} carousel - HTMLElement.
 ```